rudder-server-relay/SOURCES/rudder-pkg

```diff
@@ -56,7 +60,6 @@ COMPATIBILITY_FILE = DB_DIRECTORY + '/compatible.json'
 
 # Plugins specific resources
 PLUGINS_CONTEXT_XML = "/opt/rudder/share/webapps/rudder.xml"
-PLUGINS_JAR_DIR = '/opt/rudder/share/plugins'
```

Comment:
Does that mean that `jar-files` in metadata must contain the full path of the jar?